src/containers/facility/FacilityDetailImages.tsx

```diff
@@ -8,24 +11,32 @@ const DetailImage = styled(Image)`
   object-fit: cover;
 `;
 
-export const FacilityDetailImages = ({ images }) => {
-  return (
-    <Box
-      direction="row"
-      style={{
-        gap: '8px',
-        flex: '50%'
-      }}
-    >
-      <Box direction="column" width="100%" style={{ gap: '8px' }}>
-        <DetailImage src={images[0]?.url} />
-        <DetailImage src={images[1]?.url} />
-      </Box>
+interface FacilityDetailImagesProps {
+  images: MediaItem[];
+}
+
+export const FacilityDetailImages = forwardRef<HTMLDivElement, FacilityDetailImagesProps>(
+  ({ images }, ref) => {
+    const theme = useTheme();
+    return (
+      <Box
+        direction="row"
+        style={{
+          gap: theme.spacing(1),
+          flex: '50%'
+        }}
+        ref={ref}
+      >
+        <Box direction="column" width="100%" style={{ gap: '8px' }}>
+          {images[0] && <DetailImage src={images[0]?.url} />}
```

Comment:
a fallback image would eventually be nice, altough we can do that in a next iteration